node/core/pvf/common/src/worker.rs

```diff
@@ -309,4 +312,70 @@ pub mod thread {
 			Some(*result.0)
 		}
 	}
+
+	#[test]
+	fn get_condvar_should_be_pending() {
```

Comment:
It is best practice to put tests in their own module, so e.g. something like this. Sorry if I wasn't clear.

```rust
#[cfg(test)]
mod tests {
    use super::*;
    
    [...]
```

Reply:
You were very clear, I'm new to rust so it's my fault. I moved the test to its own module inside the thread module.
Just one question: I'm testing functions outside the thread module (e.g. cpu_time_monitor_loop) in this test. Should I create a test module outside the thread module, or is it okay the way I did it now?

Reply:
Good catch! Yes I would do this:

> Should I create a test module outside the thread module

Reply:
Just added the test module outside, thanks for your time and help